### PR TITLE
Localize all strings

### PR DESCRIPTION
**Pull request by CyberL1**
Localized all (i think) string, cuz i've been bored and want to help nertivia a little

### CODE REVIEW

**Comment by Toby222** [comment]
I think you might need to re-merge the master branch, there's several regressions in this PR, like downgrading the notifications and removing the spoiler markdown

**Comment by CyberL1** [comment]
Ok, updated to newet version of Nertivia

File: src/components/app/Popouts/Popouts/SettingsPanels/survey.vue [hunk 1]
```diff
@@ -159,7 +159,7 @@ export default {
       if (!continent) return null;
       return [
         ...this.surveyItems.countries.filter(c => c.code === continent.code),
-        ...[{ emoji: "😶", name: this.$t("rather-not-say"), code: "no" }]
+        ...[{ emoji: "😶", name: "Rather not say", code: "no" }]
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/survey.vue]: Was this reverted on purpose?

File: src/components/app/MainNav.vue [hunk 2]
```diff
@@ -235,22 +234,6 @@ export default {
...
-      return allNotifications.filter(c => {
-        const matchChannelID = c.channelID === currentChannelID;
-        if (matchChannelID && currentTab === 2 && this.focused) {
-          return false;
-        } else {
-          return true;
-        }
-      });
-    },
     serverNotification() {
```
Comment (Toby222) [inline src/components/app/MainNav.vue:237]: Reverses published changes

File: src/components/app/MainNav.vue [hunk 3]
```diff
@@ -259,13 +242,15 @@ export default {
...
             this.currentTab !== 2)
         );
       });
       const mentioned = notificationsFiltered.find(m => m.mentioned);
-      if (mentioned) return true;
-      return false;
+      return {
+        notification: !!notificationsFiltered.length,
+        mentioned: !!mentioned
+      };
```
Comment (Toby222) [inline src/components/app/MainNav.vue:253]: Reverts published changes

File: src/components/app/CustomStatus.vue [hunk 4]
```diff
@@ -6,13 +6,13 @@
       @keydown="keyDown"
       ref="textArea"
       v-if="showTextArea"
-      placeholder="Click to add status."
+      :placeholder="$t('add-user-status')"
```
Comment (Toby222) [inline src/components/app/CustomStatus.vue:9]: is the colon on purpose?

File: src/components/app/Navigation.vue [hunk 5]
```diff
@@ -13,7 +13,7 @@
             servers[toolTipServerID] &&
             servers[toolTipServerID].verified
         "
-        title="Verified Server"
+        :title="$t('verified-server')"
```
Comment (Toby222) [inline src/components/app/Navigation.vue:16]: Colon

File: src/components/app/Popouts/Popouts/AddServer.vue [hunk 6]
```diff
@@ -33,7 +33,7 @@
                 class="input"
                 type="text"
                 v-model="serverName"
-                name="Server Name"
+                :name="$t('server-name')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/AddServer.vue:36]: Colon

File: src/components/app/Popouts/Popouts/DrawPopout.vue [hunk 7]
```diff
@@ -51,28 +51,28 @@
...
         @click="deleteButton"
       >
         delete
       </div>
       <div
         class="button material-icons"
         @click="sendButton"
-        :title="currentChannelID ? 'Send' : 'Select a channel to send'"
+        :title="currentChannelID ? this.$t('send') : this.$t('send-no-channel')"
         :class="{ disabled: !currentChannelID }"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/DrawPopout.vue:76]: Colons

File: src/components/app/Popouts/Popouts/ServerSettingsPanels/General.vue [hunk 8]
```diff
@@ -51,12 +51,12 @@
...
             @input="inputEvent('name', $event)"
             :default-value="server.name"
-            name="Server Name"
+            :name="$t('server-name')"
           />
           <drop-down
             :items="channels"
             :selected-item="defaultChannel"
-            name="Default Channel"
+            :name="$t('default-channel')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/ServerSettingsPanels/General.vue:59]: Colons

File: src/components/app/Popouts/Popouts/ServerSettingsPanels/ManageChannels.vue [hunk 9]
```diff
@@ -29,13 +29,13 @@
           type="text"
           @input="inputEvent('name', $event)"
           :default-value="channels[currentChannelIndex].name"
-          name="Channel Name"
+          :name="$t('channel-name')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/ServerSettingsPanels/ManageChannels.vue:32]: Colon

File: src/components/app/Popouts/Popouts/ServerSettingsPanels/ServerVisibility.vue [hunk 10]
```diff
@@ -34,23 +33,23 @@
...
+        {{ saving ? this.$t('updating') : this.$t('update') }}
       </div>
       <div
         class="button"
         :class="{ disabled: saving }"
         v-if="!alreadyPublic && showSaveButton && !privateServer"
         @click="saveButton"
       >
-        {{ saving ? "Saving..." : "Save" }}
+        {{ saving ? "this.$t('saving') : this.$t('save') }}
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/ServerSettingsPanels/ServerVisibility.vue:52]: Misplaced quote

File: src/components/app/Popouts/Popouts/SettingsPanels/ActivityStatus.vue [hunk 11]
```diff
@@ -12,7 +10,7 @@
       selectBy="name"
       :selectedItem="0"
       :noneSelect="true"
-      name="Add A Program"
+      :name="$t('add-a-program')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/ActivityStatus.vue:13]: Colon

File: src/components/app/Popouts/Popouts/SettingsPanels/DebugSettings.vue [hunk 12]
```diff
@@ -16,12 +16,12 @@
...
   data() {
     return {
-      view: "Click on tabs above to see data."
+      view: this.$t("debug-desc")
     };
   },
   methods: {
     vuexLogButton() {
-      alert("Vuex logged to console.");
+      alert("Vuex has been logged");
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/DebugSettings.vue:24]: Reverts published change

File: src/components/app/Popouts/Popouts/SettingsPanels/EditProfile.vue [hunk 13]
```diff
@@ -73,20 +73,20 @@
           type="password"
           @input="inputEvent('new_password', $event)"
           autocomplete="new-password"
-          name="New Password"
+          :name="$t('new-password')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/EditProfile.vue:76]: Colons

File: src/components/app/Popouts/Popouts/SettingsPanels/ManageEmojis.vue [hunk 14]
```diff
@@ -52,7 +51,7 @@ export default {
   components: {},
   data() {
     return {
-      domain: config.nertiviaCDN + "/emojis/"
+      domain: config.nertiviaCDN + "emojis/"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/ManageEmojis.vue:54]: Reverts published change

File: src/components/app/Popouts/Popouts/SettingsPanels/MyThemesMakePublic.vue [hunk 15]
```diff
@@ -65,18 +64,18 @@
             >/150)
           </div>
           <textarea
-            placeholder="Description"
+            :placeholder="$t('description')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/MyThemesMakePublic.vue:67]: Colon

File: src/components/app/Popouts/Popouts/SettingsPanels/themesEditor.vue [hunk 16]
```diff
@@ -5,15 +5,15 @@
         class="theme-name"
         v-model="name"
         type="text"
-        placeholder="Theme name"
+        :placeholder="$t('theme-name')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/SettingsPanels/themesEditor.vue:8]: Colon and misplaced quotes

File: src/components/app/Popouts/Popouts/adminManageUserPopout.vue [hunk 17]
```diff
@@ -16,19 +16,19 @@
         type="password"
         autocomplete="new-password"
         v-model="password"
-        name="Confirm Password"
+        :name="$t('confirm-password')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/adminManageUserPopout.vue:19]: Colon

File: src/components/app/Popouts/Popouts/adminManageUserPopout.vue [hunk 18]
```diff
@@ -16,19 +16,19 @@
...
       />
       <custom-input
         theme="light"
         class="input"
         type="textarea"
         v-model="reason"
-        name="Suspend Reason"
+        :name="$('suspend-reason')"
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/adminManageUserPopout.vue:26]: Colon

File: src/components/app/ServerList.vue [hunk 19]
```diff
@@ -17,7 +17,7 @@
       />
       <div class="gradient" @click="bannerImageClicked" v-if="server.banner" />
       <div class="sub-banner">
-        <div class="verified" v-if="server.verified" title="Verified Server">
+        <div class="verified" v-if="server.verified" :title="$t('verified-server')">
```
Comment (Toby222) [inline src/components/app/ServerList.vue:20]: Colon

File: src/components/app/markup/MessageQuote.vue [hunk 20]
```diff
@@ -1,12 +1,12 @@
...
   <article class="quote" v-bind:class="{ quoteFailed: failMessage == null }">
     <header class="section">
-      Quoted:
+      {{ $t("quoted") }}:
       <span class="username" @click="openUserInfo">{{
         this.quote.creator.username
       }}</span>
       <i
-        title="Go to message"
+        :title="$t('go-to-message')"
```
Comment (Toby222) [inline src/components/app/markup/MessageQuote.vue:9]: Colon

File: src/components/app/markup/MessageQuote.vue [hunk 21]
```diff
@@ -23,7 +23,7 @@
     <main v-else class="fail-message">
       <!-- This quote is too long to be displayed. -->
       <Markup
-        text="This quote is too long to be displayed."
+        :text="$t('to-long-quote')"
```
Comment (Toby222) [inline src/components/app/markup/MessageQuote.vue:26]: Colon

File: src/components/app/Popouts/Popouts/ServerSettingsPanels/ServerVisibility.vue [hunk 22]
```diff
@@ -20,12 +19,12 @@
...
         <div class="input">
           <div class="title">
-            Description (<span :class="{ warn: description.length > 150 }">{{
+            {{ $t("description") }} (<span :class="{ warn: description.length > 150 }">{{
               description.length
             }}</span
             >/150)
           </div>
-          <textarea placeholder="Description" v-model="description"></textarea>
+          <textarea :placeholder="$t('description')" v-model="description"></textarea>
```
Comment (Toby222) [inline src/components/app/Popouts/Popouts/ServerSettingsPanels/ServerVisibility.vue:27]: Colon

**Review by Toby222 — CHANGES_REQUESTED** [review]
Inconsistent use between `this.$t` and `$t`, as well as `"` and `'` for strings
Also a bunch of things you missed

File: src/lang/en.json [hunk 23]
```diff
@@ -49,17 +49,309 @@
...
+    "undo": "Undo",
+    "redo": "Redo",
+    "pen-color": "Pen Color",
+    "custom-color": "Custom Color",
+    "pen-size": "Pen Size",
+    "fullscreen": "Fullscreen",
+    "send": "Send",
+    "send-no-channel": "Select a channel to send",
+
+    "GDrive-notice": "To upload files, You must link your Google Drive account with your Nertivia account.",
```
Comment (Toby222) [inline src/lang/en.json:117]: Key should be lower hypen-case for consistency

**Pull request closed without merging**